static/sass/_pattern_buttons.scss

```diff
@@ -5,7 +5,7 @@
   }
 
   .p-button--tag {
-    @extend %p-button-light;
+    @extend %p-button-light !optional;
```

Comment:
What does optional do here?

Reply:
`./run` was failing because of this line and it suggested adding `!optional` to this variable

Reply:
(just removed it as Scott pointed out it was a new variable I just needed to rebase)